### ISSUE TITLE
Listen to webWidget events

### ISSUE THREAD

**Issue by spicy-sauce** [dansmaculotte/vue-zendesk#29]
Currently (as I understand it), the only event triggered is the loaded custom one. 
There's no support for the webwidget events, close for example.
I had to add this explicitly:
```
zE("webWidget:on", "close", () => {
  console.log("Closing widget...");
  zE("webWidget", "hide");
});
```

It would be nice to throw an event once the widget is closed for example with an appropriate callback.
What do you think?

**Comment by rtouze** [comment]
Hi, thank you for your feedback. It would be a nice feature for sure, but I don't think I will add it for now.

**Comment by rtouze** [comment]
Finally, I need this feature. You'll find it in next release.

**Comment by rtouze** [comment]
Fixed in 686e333f846371cfcc3cc5b1e0d791cbf9c6d7f7

**Comment by spicy-sauce** [comment]
Thank you so much!
